### PR TITLE
Move Consumption to node

### PR DESCRIPTION
**Pull request by georgevanderson**
Additionally fixes for using pnpm and properly killing child processes of ts-node

### CODE REVIEW

File: packages/ts-moose-lib/src/commons.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+const [
+  ,
+  ,
+  CLICKHOUSE_DB,
+  CLICKHOUSE_HOST,
+  CLICKHOUSE_PORT,
+  CLICKHOUSE_USERNAME,
+  CLICKHOUSE_PASSWORD,
+  CLICKHOUSE_USE_SSL,
+] = process.argv;
```
Comment (callicles) [inline packages/ts-moose-lib/src/commons.ts]: That feels a bit risky to have in commons - that probably should say in the called script no?

File: packages/ts-moose-lib/src/commons.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      ? "https"
+      : "http";
+  console.log(
+    `Connecting to Clickhouse at ${protocol}://${CLICKHOUSE_HOST}:${CLICKHOUSE_PORT}`,
+  );
+  return createClient({
+    url: `${protocol}://${CLICKHOUSE_HOST}:${CLICKHOUSE_PORT}`,
+    username: CLICKHOUSE_USERNAME,
+    password: CLICKHOUSE_PASSWORD,
+    database: CLICKHOUSE_DB,
```
Comment (callicles) [inline packages/ts-moose-lib/src/commons.ts]: and make those parameters?

**Review by callicles — APPROVED** [review]
Nice work!

File: apps/framework-internal-app/app/flows/UserActivity/ParsedActivity/flow.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@ export default function run(event: UserActivity): ParsedActivity {
   // Convert local timestamp to UTC and return new ParsedActivity object.
   return {
     eventId: event.eventId, // Retain original event ID.
-    userId: "puid" + event.userId, // Example: Prefix user ID.
+    userId: "puid2" + event.userId, // Example: Prefix user ID.
```
Comment (DatGuyJonathan) [inline apps/framework-internal-app/app/flows/UserActivity/ParsedActivity/flow.ts]: Why puid2? This affects my load test so ideally we don't change this if we don't have to.

**Pull request merged**